nativelib/src/main/scala/scala/scalanative/native/fcntl.scala

```diff
@@ -0,0 +1,39 @@
+package scala.scalanative
+package native
+
+import stat.mode_t
+
+@extern
+object fcntl {
```

Comment:
Done! I've moved `dirent`, `fcntl`, `limits`, `stat` and `utime` to the `posix` package.